Documentation/operations/upgrade.rst

```diff
@@ -292,12 +292,16 @@ communicating via the proxy must reconnect to re-establish connections.
 
 1.18 Upgrade Notes
 ------------------
+* ``cilium-dbg bpf policy`` will parse the cilium internal identity ``1`` correctly from ``reserved:unknown`` to ``ANY``.
```

Comment:
The internal identity wasn't 1, it was `0`. Here's a slightly more user-centric way to phrase the change since I wouldn't expect users to know the internal identities anyway, what do you think?

```suggestion
* ``cilium-dbg bpf policy`` now prints ``ANY`` and not ``reserved:unknown`` for a bpf policy entry that allows any peer identity.
```

Reply:
yes. that's definitely better and thanks for pointing it out